src/main/java/reposense/model/AuthorConfiguration.java

```diff
@@ -145,10 +145,21 @@ public void addAuthor(Author author, List<String> ignoreGlobList) {
     public void removeIgnoredAuthors(List<String> ignoredAuthorsList) {
         for (String author : ignoredAuthorsList) {
             if (authorEmailsAndAliasesMap.containsKey(author)) {
-                authorList.remove(authorEmailsAndAliasesMap.get(author));
+                Author ignoredAuthor = authorEmailsAndAliasesMap.get(author);
+                List<String> aliases = new ArrayList<>(ignoredAuthor.getAuthorAliases());
+                removeIgnoredAuthorsInformation(author, aliases, ignoredAuthor);
             }
         }
-        resetAuthorInformation();
+    }
+
+    /**
+     * Removes information of the {@code author} along with its {@code aliases}
+     */
+    public void removeIgnoredAuthorsInformation(String author, List<String> aliases, Author ignoredAuthor) {
+        authorList.remove(ignoredAuthor);
+        authorDisplayNameMap.remove(ignoredAuthor);
+        authorEmailsAndAliasesMap.remove(author);
+        aliases.forEach(alias -> authorEmailsAndAliasesMap.remove(alias, ignoredAuthor));
```

Comment:
Related to the test failure: 

- I think we'll need to remove the key-value bindings for ignoredAuthors' emails as well
- Is there a guarantee that the `author` parameter == the git ID of the `ignoredAuthor`? From the code, it seems `author` could actually be the email or an alias or ignoredAuthor. If that's the case, we need to remove the key-value binding for ignoreAuthor's git ID

- Actually, do we need the `author` parameter at all? I think all the info needed to be removed (aliases, emails, git id) is available in `ignoredAuthor` right?